src/main/java/fi/pohina/vinkkilista/App.java

```diff
@@ -1,16 +1,30 @@
 package fi.pohina.vinkkilista;
 
+import spark.ModelAndView;
+import java.util.HashMap;
+import java.util.Map;
+import spark.template.thymeleaf.ThymeleafTemplateEngine;
 import static spark.Spark.*;
 
 public class App {
     public static void main(String[] args) {
         port(getPort());
 
         get("/", (req, res) -> {
-            return "Hello world!";
+            Map<String, Object> map = new HashMap<>();
+            return render(map, "index");
```

Comment:
You could use the `ModelAndView` class from spark instead. It would look something like this:
```java
get("/", (req, res) -> {
  HashMap<Integer, String> map = new HashMap<>();
  return ModelAndView(map, "index");
}, new ThymeleafTemplateEngine())
```

Reply:
Sparkjava suggests the current style: http://sparkjava.com/documentation#views-and-templates